x-pack/plugins/observability/common/apm_ui_experimental_features_settings.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { schema } from '@kbn/config-schema';
```

Comment:
This is likely it. You can't import concrete things from `@kbn/config-schema` to common code, as it will bundle all config-schema and its dependencies to the client-side bundle.

Reply:
@pgayvallet hmm that's too bad, any way to overcome it? I'd like to have the APM settings defined in a common place because I use it to build the UI.

Reply:
There isn't really a way to do that artificially AFAIK. I'm afraid the only option here is to only keep in `common` the subset of the `apmExperimentalFeaturesSettings` fields that are used both on the browser and server-side, and to extend this data with the server only fields (like the schema) in server-side only code.

Reply:
I fixed it by creating an API on the APM server that provides the experimental features available in the UI. So, there's no reason to break these settings into two files.